arq/utils.py

```diff
@@ -26,7 +30,23 @@ def to_unix_ms(dt: datetime) -> int:
 
 
 def ms_to_datetime(unix_ms: int) -> datetime:
-    return datetime.fromtimestamp(unix_ms / 1000, tz=timezone.utc)
+    """
+    convert milliseconds to datetime, use the timezone in os.environ
+    """
+    tz = None
+    for timezone_key in timezone_keys:
+        tz = os.environ.get(timezone_key)
+        if tz:
+            break
+
+    if tz is not None and isinstance(tz, str):
+        try:
+            tz = pytz.timzone(tz)
+        except UnknownTimeZoneError:
+            tz = timezone.utc
```

Comment:
this error should be raised, or at least warned about. Silently catching errors like this is very confusing.